CHANGELOG.md

```diff
@@ -2,15 +2,24 @@
 
 ## Unreleased
 
-### Build tool
+### Language Server
```

Comment:
Could you put back the other headings and rebase on main to resolve the conflict please 🙏

Reply:
Done :)